projects/tabs/src/modules/tabs/tab-button.component.ts

```diff
@@ -60,8 +60,23 @@ export class SkyTabButtonComponent {
     }
   }
 
-  public onButtonEnterKeyDown(event: any): void {
-    this.onButtonClick(event);
+  public onTabButtonKeyDown(event: KeyboardEvent): void {
+    /*istanbul ignore else */
+    if (event.key) {
+      switch (event.key.toUpperCase()) {
+        case ' ':
+        case 'ENTER':
+          /*istanbul ignore else */
+          if (!this.disabled) {
```

Comment:
Why are you using this internal if-block here instead of just calling into `onButtonClick` like we where before?

Reply:
Reverted.